### ISSUE TITLE
Background blur is not proper Gaussian blur

### ISSUE THREAD

**Issue by szaszm** [sociallydistantgame/sociallydistant#9]
Background blur looks wrong in SD. I made a side-by-side comparison with GIMP, using X and Y size of 4 (based on SD source code), and the game with and without blur.

![sd_blur_bug_sbs](https://github.com/sociallydistantgame/sociallydistant/assets/1170582/0e2efc8e-0630-4e68-a999-def07eadd270)

**Comment by WinFan3672** [comment]
+1

**Comment by acidiclight** [comment]
I am concerned about the darkness in the GIMP blur. Floating Window Overlays (the background of System Settings) render a translucent rectangle overtop of the background blur, the darkening of the background isn't part of the blur itself. This is so that the game can still render the overlay even when the user has background blurs disabled.

**Comment by szaszm** [comment]
I blurred your screenshot (from Discord) of the darkened settings background before background blur was implemented on the new MonoGame engine, that's why it's darker. I think the order of darkening and blurring shouldn't matter much.

**Comment by acidiclight** [comment]
I'm in the middle of porting Terminal to `work/monogame`. I'm almost done. I'll look into the blur shader once Terminal's working, since it's another widget that uses background blurs.

**Comment by acidiclight** [comment]
Above commit should address this issue and resolve it.

Obviously, I'm blind, and can't verify - but we did have a **lengthy** discussion on the Discord server about re-implementing the background blur. Mostly me being annoyed at and incompetent with math. I've posted screenshots and a few people said it looks correct.

System Settings with darkening overlays:
![image](https://github.com/sociallydistantgame/sociallydistant/assets/92193769/295ad06a-4ed8-4abd-98a2-e7cbcf0a25ea)

Terminal with its char-cell renderer disabled:
![image](https://github.com/sociallydistantgame/sociallydistant/assets/92193769/4e563b72-1e54-47dc-937b-b7121850611b)

Same terminal but with System Settings opened, without the darkening overlay:
![image](https://github.com/sociallydistantgame/sociallydistant/assets/92193769/b5172e6a-84c1-4b3a-ae6e-ad3e9ad24898)


System Settings should be blurring the background as well, so the Terminal area should appear more blurry when System Settings opens. Can anyone confirm that's the case in those screenshots? If so, and if the Gaussian implementation is correct, I'll close this as resolved.

> [!NOTE]
> These screenshots are from the MonoGame build of the game, so I wouldn't compare them to Unity screenshots.

> [!WARNING]
> I like writing these fancy note boxes

> [!NOTE]
> They're colorful
